opm/input/eclipse/Schedule/VFPProdTable.cpp

```diff
@@ -538,9 +538,10 @@ Dimension VFPProdTable::ALQDimension(const ALQ_TYPE& alq_type, const UnitSystem&
     case ALQ_TYPE::ALQ_UNDEF:
         break;
     case ALQ_TYPE::ALQ_PUMP:
+        break;
     case ALQ_TYPE::ALQ_COMP:
```

Comment:
Why is this break added? I think we are suppose to throw if input is ALQ_PUMP, ALQ_COMP or ALQ_BEAN

Reply:
that's precisely why i asked you to look at it. The bug, ie, the missing throw, meant we have fell through to the return earlier. When I fix the throw, the model4 regression tests fail because they use PUMP.

Reply:
@bska or @GitPaean Do we support ALQ_PUMP? What is it???

Reply:
> Do we support ALQ_PUMP?

Not really, but we have to do **something** here lest the MODEL4 tests break due to [using `PUMP`](https://github.com/OPM/opm-tests/blob/51bdd839c2b28256542078920e9d95602304df41/model4/include/vfp_opu1.ecl#L12-L14) in its `VFPPROD` curves.

> What is it???

It's a "pump rating".  Effectively a turning frequency (number of revolutions per second), given in Hz.  The scaling factor is, therefore, one (1).

Reply:
So then the break should be fine.